base-notebook/Dockerfile

```diff
@@ -26,24 +26,24 @@ USER root
 # Conda version
 ARG conda_version="4.9.2"
 # Miniforge installer patch version
-ARG miniforge_patch_number="5"
+ARG miniforge_patch_number="7"
 # Miniforge installer architecture
 ARG miniforge_arch="x86_64"
 # Python implementation to use 
 # can be either Miniforge3 to use Python or Miniforge-pypy3 to use PyPy
```

Comment:
Let's update these lines to include Mambaforge info?

Reply:
Comment updated, good suggestion.